src/main/webapp/qaReport/NoIrbTable.js

```diff
@@ -0,0 +1,47 @@
+import { Component } from 'react';
+import { TableComponent } from "../components/TableComponent";
+import {
+  columns,
+  COLUMNS_TO_HIDE_FROM_EXCEL,
+  defaultSorted,
+  NO_IRB,
+  SIZE_PER_PAGE_LIST
+} from "../util/QaReportConstants";
```

Comment:
nit: indentation is different from `IrbTable.js`. I suggest you pick one and be consistent.